app/src/main/res/values/dimens.xml

```diff
@@ -12,6 +12,6 @@
   <dimen name="weather_icon_width">150dp</dimen>
   <dimen name="weather_icon_height">150dp</dimen>
 
-  <dimen name="face_recognition_text">100sp</dimen>
+  <dimen name="face_recognition_text">50sp</dimen>
```

Comment:
Yeah, these are just a placeholder for the final graphic (if we will have one in time 😂 )